.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
```

Comment:
Can we remove the dependabot for now?

The repository is maintained but we don't have many changes and for such a project these PRs usually just stay open and cause email spam

Reply:
sure